src/qhttp/AjaxEndpoint.h

```diff
@@ -58,7 +58,9 @@ class AjaxEndpoint
 
 private:
 
-    AjaxEndpoint();
+    AjaxEndpoint(std::shared_ptr<Server> server);
```

Comment:
The parameter needs to be defined as:
```
std::shared_ptr<Server> const& server
```

Reply:
Hmm... Passing `shared_ptr` by `const&` is perennially controversial (see https://stackoverflow.com/questions/3310737/should-we-pass-a-shared-ptr-by-reference-or-by-value and many others).  I have tended to favor the "take shared_ptr by value if you intend to store and share" guideline.  I will poll the group and we'll go with the tyranny of the majority :-)

Reply:
Per discussion in db team room, went with pass by value here and added `std::move` where ctor takes ownership of the `shared_ptr` (did same systematically through the rest of the `qhttp` code as well)

Reply:
👍